core/README.md

```diff
@@ -0,0 +1,31 @@
+# Chainlink Core
+
+This directory contains the majority of the code for the backend of Chainlink.
+
+Outside of the code contained in this repo, some static assets are pulled in using
+[packr](https://github.com/gobuffalo/packr), in the sibling directory `packr/`.
+For the backend, the solidity ABIs are used to generate types for interacting with
+Ethereum contracts. Additionally, the static assets generated by compiling the
+sibling directroy `operator-ui/` are built by packr and included in the final
+binary.
+
+See the [root README](https://github.com/smartcontractkit/chainlink/blob/master/README.md#install)
+for instructions on how to build the full Chainlink node.
+
+## Common Commands
+
+**Install:**
+
+By default `go install` will install this directory under the name `core`.
+You can instead, build it, and place it in your path as `chainlink`:
+```
+go build -o chainlink-core .
+cp chainlink-core $GOPATH/bin/chainlink
```

Comment:
```suggestion
mv chainlink-core $GOPATH/bin/chainlink
```
Then we can remove the next line (`rm ./chainlink-core`).

Reply:
What about a one liner?
```
go build -o $GOPATH/bin/chainlink .
```